src/Symfony/Bridge/Twig/NodeVisitor/Scope.php

```diff
@@ -52,7 +52,7 @@ public function enter()
     /**
      * Closes current scope and returns parent one.
      *
-     * @return Scope|null
+     * @return self|null
```

Comment:
I'm not sure we should prefer using `self` over the class/interface name. IMHO, we should only distinguish `$this`, `static` and `classname` usages, but `self` does not bring anything more, so it isn't worth it.